changelog.md

```diff
@@ -2,4 +2,6 @@
 
 2022-04-27 - 142465669cff13509290ca84ea9914cdfad613b8 - AutocompleteInput.vue - cleared options each on every search initiation to refrain from showing old list items untill the API request resolves
 
-2022-05-31 - 47b230b2ae9fad2a68033c6de25a0dba048416f0 - Divider.vue - Applied in BuzzShareViewer.vue, BuzzCreatePost.vue, BuzzFileUploadModal.vue, BuzzImageViewer.vue, BuzzPost.vue, CommentList.vue, KudosPoost.vue, VideoPost.vue, CandidateProfile.vue, AddNewFolderDialogue.vue, AddNewReportDialogue.vue, ReportsCatalogue.vue in the codebase.
\ No newline at end of file
+2022-05-31 - 47b230b2ae9fad2a68033c6de25a0dba048416f0 - Divider.vue - Applied in BuzzShareViewer.vue, BuzzCreatePost.vue, BuzzFileUploadModal.vue, BuzzImageViewer.vue, BuzzPost.vue, CommentList.vue, KudosPoost.vue, VideoPost.vue, CandidateProfile.vue, AddNewFolderDialogue.vue, AddNewReportDialogue.vue, ReportsCatalogue.vue in the codebase.
+
+2022-06-03 - e45f6bee3abf87ffc71b9265746d2d372a53253f - Label.vue - Explicitly set .oxd-label margin to zero, since it's getting overridden by angular application styles.
```

Comment:
Better to change the order of the change logs. Most recent change at the top.

Reply:
Changed to have most recent at top